webapi/src/main/scala/org/knora/webapi/slice/ontology/repo/service/OntologyRepoLive.scala

```diff
@@ -139,6 +141,26 @@ final case class OntologyRepoLive(private val converter: IriConverter, private v
     }
   }
 
+  def knoraApiRepresentationClassIriFor(classIri: ResourceClassIri): Task[ResourceClassIri] = {
+    val representations = Seq(
+      KB.StillImageRepresentation,
+      KB.MovingImageRepresentation,
+      KB.AudioRepresentation,
+      KB.ArchiveRepresentation,
+      KB.DocumentRepresentation,
+      KB.TextRepresentation,
+    )
+
+    for {
+      superClassIris <- findDirectSuperClassesBy(classIri.toInternalIri).map(_.map(_.entityInfoContent.classIri))
```

Comment:
question: does `findDirectSuperClassesBy` include multiple layers of inheritance? or is it just one level, as the name suggests? If so, I think it would be worth following up on that: A sub-class of a sub-class of a StillImageRepresentation is still a StillImageRepresentation after all.